region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -137,6 +137,55 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Morph Ball Dodge)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        "Morph"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 17,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Open the door in advance, then lure the Sidehoopers to the left corner with morph. Wait for their jumps to be mostly synchronized.",
+        "Roll right as they start a jump, then quickly unmorph and start running right.",
+        "With good timing, Samus should be able to start running a couple tiles left of the leftmost Yapping Maw."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Tank the Damage)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        {"enemyDamage": {
+          "enemy": "Sidehopper",
+          "type": "contact",
+          "hits": 1
+        }},
+        {"or": [
+          "Morph",
+          {"enemyDamage": {
+            "enemy": "Sidehopper",
+            "type": "contact",
+            "hits": 3
+          }},
```

Comment:
Without morph, horizontal damage boost + a thorn hit + 1 (additional) sidehopper hit is probably fine

Reply:
as a 3rd option you mean? You jump into the ceiling and dboost to get to the left wall?  Trying to visualize this.  My 3 hits was just running around like an idiot luring them to the left.